src/javascript/app_2/App/Components/Layout/Header/account_info.jsx

```diff
@@ -0,0 +1,28 @@
+import PropTypes    from 'prop-types';
+import React        from 'react';
+import { localize } from '../../../../../_common/localize';
+
+const AccountInfo = ({ balance, currency, loginid }) => (
+    <div className='acc-balance'>
+        <div className='acc-info'>
+            <p className='acc-balance-currency'>{`${(currency || '').toUpperCase()} ${localize('Account')}`}</p>
+            <p className='acc-balance-accountid'>{loginid}</p>
+        </div>
+        { typeof balance !== 'undefined' &&
+            <p className='acc-balance-amount'>
+                <i><span className={`symbols ${(currency || '').toLowerCase()}`}/></i>
+                {balance}
+            </p>
+        }
+    </div>
+);
+
+AccountInfo.propTypes = {
+    balance       : PropTypes.string,
+    can_upgrade   : PropTypes.bool,
+    currency      : PropTypes.string,
+    loginid       : PropTypes.string,
+    onClickUpgrade: PropTypes.func,
```

Comment:
Seems that `onClickUpgrade` and `can_upgrade` not used in this component.

Reply:
thanks 👍